code/network/multi_fstracker.cpp

```diff
@@ -522,6 +522,22 @@ int multi_fs_tracker_inited()
 	return Multi_fs_tracker_inited;
 }
 
+// if the PXO is actively being used (logged in)
+bool multi_fs_tracker_in_use() 
+{
+	// if on the standalone, we're either PXO or not, so just check if it's been initalized.
+	if (Game_mode & GM_STANDALONE_SERVER) {
+		return multi_fs_tracker_inited();
+	}
+
+	// on regular servers, the flag is set in the options menu. 
+	if (Multi_options_g.pxo) {
+		return multi_fs_tracker_inited();
+	}
```

Comment:
Uh, both of these if() blocks call the same function.

Reply:
And `Multi_options_g.pxo` is used on both standalone and regular servers, so it's extra redundant.

Probably better to just modify `multi_fs_tracker_inited()` like so:
`return Multi_options_g.pxo && Multi_fs_tracker_inited;`

It would do the same thing and not require an extra function.